setup.py

```diff
@@ -677,10 +678,11 @@ def srcpath(name=None, suffix='.pyx', subdir='src'):
     obj = Extension('pandas.{name}'.format(name=name),
                     sources=sources,
                     depends=data.get('depends', []),
-                    include_dirs=include,
+                    include_dirs=include + [numpy.get_include()],
                     language=data.get('language', 'c'),
                     define_macros=data.get('macros', macros),
-                    extra_compile_args=extra_compile_args)
+                    extra_compile_args=['-fopenmp'] + extra_compile_args,
+                    extra_link_args=['-fopenmp'])
```

Comment:
is this platform-dependent in any way?  i.e. do we need to check that it is available?

Reply:
I believe this is the gcc flag to link against openmp and I think clang uses the same flag but I'm not sure.